### PR TITLE
scorecard:Add support for custom scorecard

### PR DESCRIPTION
**Pull request by avisiedo**
- It adds minimal changes to start to get a custom scorecard container image and run it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,6 @@ dive.*.tar
 
 # Ignore .venv directory that is used for python environment
 .venv
+
+# Ignore generated scorecard property file
+config/scorecard/scorecard.properties
```
Comment (avisiedo) [inline .gitignore:55]:
```suggestion
config/scorecard/scorecard.properties
```

Comment (avisiedo) [inline .gitignore]: nit: Add new line to the end of file

**Comment by avisiedo** [comment]
Update github actions to build and publish the custom scorecard image.

**Review by vashirov — APPROVED** [review]
/lgtm

**Pull request merged**